### PR TITLE
ENG-3554 add build --all-ms command

### PR DESCRIPTION
**Pull request by alepintus**

### CODE REVIEW

File: src/commands/build.ts [hunk 1]
```diff
@@ -11,30 +13,46 @@ export default class Build extends BaseBuildCommand {
...
       name: 'name',
-      description: 'The name of the component to build',
-      required: true
+      description: 'The name of the component to build'
     }
   ]
 
+  static flags = {
+    'all-ms': Flags.boolean({
+      description: 'Build all the bundle microservices'
```
Comment (cecchisandrone) [inline src/commands/build.ts]: Builds

File: src/commands/build.ts [hunk 2]
```diff
@@ -1,7 +1,9 @@
-import { CliUx } from '@oclif/core'
+import { CliUx, Flags } from '@oclif/core'
 import { BundleService } from '../services/bundle-service'
 import { ComponentService } from '../services/component-service'
 import { BaseBuildCommand } from './base-build'
+import { Phase } from '../services/command-factory-service'
+import { ComponentType } from '../models/component'
 
 export default class Build extends BaseBuildCommand {
   static description = 'Build bundle components'
```
Comment (cecchisandrone) [inline src/commands/build.ts]: Builds

File: src/commands/build.ts (hunk 1 above)
Comment (alepintus) [reply src/commands/build.ts]: done

File: src/commands/build.ts (hunk 2 above)
Comment (alepintus) [reply src/commands/build.ts]: done

**Review by cecchisandrone — APPROVED** [review]

**Pull request merged**